### ISSUE TITLE
Subscriber form: CTA accessibility improvements

### ISSUE THREAD

**Issue by bogiii** [Automattic/wp-calypso#68389]
CFT Feedback:

The subscriber import button is not labeled accessibly. The button currently reads to a screen reader as “Or bring your mailing list from other newsletter services by”. This should probably have an ARIA label attached or otherwise indicate what the button actually says.
